src/main/java/com/familytime/model/entity/Task.java

```diff
@@ -0,0 +1,386 @@
+/**
+ * Created by Andrii Gaidychuk on 18.09.2016.
+ */
+
+package com.familytime.model.entity;
+
+import com.familytime.model.serializer.JsonDataTimeSerializer;
+import com.familytime.model.serializer.JsonDateSerializer;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.springframework.beans.factory.annotation.Value;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+/**
+ * Class for reflect table tast from persistence layout
+ *
+ * @version 1.0
+ */
+
+@SuppressWarnings("serial")
+@Entity
+@Table(name = "task")
+public class Task implements Serializable {
+
+    /// *** Properties  *** ///
+
+    @Id
+    @GeneratedValue
+    @Column(name = "id")
+    protected Long id;
+
+    @NotNull
+    @ManyToOne
+    @JoinColumn(name = "task_type_id")
+    protected TaskType taskType;
+
+    @NotNull
+    @ManyToOne
+    @JoinColumn(name = "complexity_id")
+    protected Complexity complexity;
+
+    @NotNull
+    @ManyToOne
+    @JoinColumn(name = "creator_id")
+    protected User creator;
+
+    @NotNull
+    @ManyToOne
+    @JoinColumn(name = "performer_id")
+    protected User performer;
+
+    @NotNull
+    @ManyToOne
+    @Column(name = "bonus_id")
+    protected Bonus bonus;
+
+    @NotNull
+    @Value("false")
+    @Column(name = "has_subtasks")
+    protected Boolean hasSubtasks;
+
+    @ManyToOne
+    @JoinColumn(name = "parent_id")
+    protected Task task;
+
+    @Column(name = "step")
+    protected Integer step;
+
+    @Column(name = "description")
+    protected String description;
+
+    @NotNull
+    @Column(name = "prize")
+    protected Integer prize;
+
+    @NotNull
+    @JsonSerialize(using = JsonDataTimeSerializer.class)
+    @Column(name = "close_to")
+    protected LocalDateTime closeTo;
+
+    @JsonSerialize(using = JsonDataTimeSerializer.class)
+    @Value("#{LocalDateTime.now}")
```

Comment:
@andrycreator It would be better to use JPA for setting current date&time.
Use JPA annotation `@PrePersist`.